data/monster/Sorcerers/incredibly_old_witch.xml

```diff
@@ -1,83 +1,80 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<monster name="Incredibly Old Witch" nameDescription="an incredibly old witch" race="blood" experience="0" speed="180">
-	<health now="1" max="1" />
-	<look type="54" head="20" body="30" legs="40" feet="50" corpse="6081" />
+<monster name="Incredibly Old Witch" nameDescription="an incredibly old witch" race="blood" experience="0" speed="202">
+	<health now="100" max="100" />
+	<look type="54" corpse="20535" />
 	<targetchange interval="5000" chance="8" />
 	<flags>
 		<flag summonable="0" />
 		<flag attackable="1" />
 		<flag hostile="1" />
-		<flag illusionable="1" />
+		<flag illusionable="0" />
```

Comment:
as far as I remember incredibly old witch is illusionable, if you walk up to her and she casts the AOE changer spell on herself it will change her to the animal too.

Reply:
illusionable flag is the one about the spell "utevo res monster name" that will transform you into a rag 😄  the one you want to mean is the immunity flag outfit 😋